### PR TITLE
added regression test question to the qa quiz

### PR DESCRIPTION
**Pull request by alondramora**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/quality-assurance-quiz.ts [hunk 1]
```diff
@@ -1,5 +1,15 @@
 const qualityAssuranceQuiz = [
-   {
+  {
+    Question: "What type of testing is done on a new build to ensure that new functionality has not broken previously tested functionality?",
```
Comment (jdwilkin4) [inline src/data/quality-assurance-quiz.ts:3]:
```suggestion
    Question: "What type of testing is done on a new feature to ensure that it doesn't accidentally break existing functionality?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/quality-assurance-quiz.ts (hunk 1 above)
Comment (alondramora) [reply src/data/quality-assurance-quiz.ts:3]: Thank you for the suggestion on the question, it sounds a lot better that way. I will commit the suggestion now. 😊

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**